documentation-site/pages/discover-more/versioning-policy.mdx

```diff
@@ -11,22 +11,27 @@ export default Layout;
 
 # Base Web Versioning Policy
 
-We recognize that you need stability from Base Web.
+**We recognize that you need stability from Base Web.**
 
-This document describes the practices that we follow to provide you the Base Web library.
-We want everyone who depends on Base Web to know when and how new features are added,
-and to be well-prepared when obsolete ones are removed.
+This document describes the practices that we follow to provide you the Base Web library. We want everyone who depends on Base Web to know when and how new features are added and to be well-prepared when obsolete ones are removed.
 
-[Please visit the release history](https://github.com/uber/baseweb/releases). This log
-includes a list of bug fixes and new features, as well as breaking changes and migration guides.
+[Please visit the release history](https://github.com/uber/baseweb/releases) for previous releases. This log includes a list of bug fixes and new features, as well as breaking changes and migration guides.
 
-The project follows [SemVer](https://semver.org), with a few flavours. We do not
+## SemVer, with a Few Flavours
+
+The project follows [SemVer](https://semver.org), with a few flavors. We do not
 bump major versions for the following changes:
 
 - Any component or function that's prefixed with `Unstable_` / `unstable_` may change or be removed without notice.
 - Visual changes, like colors. Under special circumstances, like sweeping visual changes, we may release a major version.
 - Change in undocumented APIs and internal data structures.
 
+## Gradual Upgrades via Warnings
+
 _Development builds of Base Web include many helpful warnings._
 
-In the event of a future breaking change, we will add warnings which indicate the feature is scheduled for deprecation and will provide migration instructions. Deprecation warnings may be added in a minor version. Following major versions will remove the deprecated feature.
+In the event of a future breaking change, we will add warnings that indicate the feature is scheduled for deprecation and will provide migration instructions. Deprecation warnings may be added in a minor version. The following major versions will remove the deprecated feature.
+
+## Breaking Changes
+
+Breaking changes are inconvenient for everyone, so we thrive to **limit the number of breaking changes by releasing only one major version a year, around September.** To help with the migration, we provide codemods for every major release. We may release major versions at any time in cases of unforeseen emergencies.
```

Comment:
@chasestarr @tajo @nadiia @sandgraham what do you think about this? do you think it's realistic?

Reply:
This sounds good to me

Reply:
I'm a little worried that the one major a year will end up as a [christmas tree major](https://en.wikipedia.org/wiki/Christmas_tree_bill), with lots of unrelated breakages included. Is that better than several smaller majors with specific changes?

Reply:
it seems like most people prefer that - doing one bigger migration a year, then a few smaller

at least, that's what the feedback we got was

Reply:
with that said, I've added a disclaimer, that we may release major versions anytime, if we think it's needed